src/react-native/04-intro-to-jsx/intro-to-jsx.md

```diff
@@ -12,7 +12,7 @@ In this section, you will:
 
 - TODO
 
-## Objective 1: TODO
+## Objective 1: Variables
```

Comment:
React Native is going to be different from TypeScript in that we want the objective titles to match what the user is trying to accomplish while building the app.

Feel free to just copy what we have in the React course if it makes sense: https://github.com/bitovi/academy/blob/b465c568da18defa464ea136798f479eb0830eb5/src/react-vite/03-intro-to-jsx/intro-to-jsx.md#L18